src/box/iproto.cc

```diff
@@ -3482,6 +3619,122 @@ iproto_thread_init(struct iproto_thread *iproto_thread)
 	rlist_create(&iproto_thread->connections);
 }
 
+/**
+ * Returns `enum iproto_type' if `name' is a valid name of the IPROTO overriding
+ * event, or iproto_type_MAX otherwise. If the name contains correct prefix, but
+ * the request type is invalid, the error is logged with CRIT log level.
+ * `is_by_id' set to True if the request is overridden by id, False if by name.
+ */
+static uint32_t
+get_iproto_type_from_event_name(const char *name, bool *is_by_id)
+{
+	*is_by_id = false;
+	const char *prefix = "box.iproto.override";
+	const size_t prefix_len = strlen(prefix);
+	if (strncmp(name, prefix, prefix_len) != 0)
+		return iproto_type_MAX;
```

Comment:
I'm not sure it's a good decision because in future we can have a request type with value 129 (current `iproto_type_MAX` value).
I think we shouldn't have any reserved values here - let's return bool indicating if event name was successfully parsed and return value with an output argument.
Also, let's test this case if we can.

Reply:
Done.

> Also, let's test this case if we can.

Added a testcase with hardcoded value 129.